scio-test/src/test/scala/com/spotify/scio/coders/CoderTestUtils.scala

```diff
@@ -36,4 +45,13 @@ object CoderTestUtils {
     result == value
   }
 
+  def testRoundTrip[T: ClassTag](ser: Serializer[T],
+                                 elems: T,
+                                 k: Kryo = KryoSerializer.registered.newKryo(),
+                                 bufferSize: Int = 1024): Unit = {
+    val o = ByteBuffer.allocate(bufferSize).array()
+    ser.write(k, new Output(o), elems)
+    elems === ser.read(k, new Input(o), ScioUtil.classOf[T])
```

Comment:
Is `===` equivalent to `a should equal b`? I prefer to spell it out for clarity.
Also this is technically a `SerializerTestUtil` and not for `Coder[T]`. Since it's used in only one class, keep it in `JIterableWrapperSerializerTest` until there's a need to split?